### PR TITLE
Fix import error & requirements.txt

### PR DESCRIPTION
**Pull request by namirinz**
Fix requirements.txt
- add python-dotenv package
- remove package version

FIx main.py
- change import

### CODE REVIEW

**Comment by namirinz** [comment]
@phusitsom  @jiratQ

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
...
-discord.py==1.7.3
-gTTS==2.2.3
-langdetect==1.0.9
-PyNaCl==1.4.0
+discord
+discord.py
+gTTS
+langdetect
+PyNaCl
+python-dotenv
```
Comment (phusitsom) [inline requirements.txt:6]: Good

File: main.py [hunk 2]
```diff
@@ -8,9 +8,9 @@
 from src.utils.utils import guess_command
 from src.utils.kick import random_kick
 
-from party import get_players
-from utils import guess_command
-from utils import config
+from src.utils.party import get_players
+from src.utils.utils import guess_command
+from src.utils.utils import config
```
Comment (phusitsom) [inline main.py:13]: OK

**Review by phusitsom — APPROVED** [review]
Approved

**Review by jiratQ — APPROVED** [review]
**very clean code
S U P E R good**

**Pull request merged**